src/main/java/ListCommand.java

```diff
@@ -0,0 +1,21 @@
+public class ListCommand extends Command {
```

Comment:
I like that you managed to encapsulate all your commands under a single Command class

Reply:
Thanks for the input.